tests/beaker_tests/vxlan_global/test_vxlan_global.rb

```diff
@@ -0,0 +1,222 @@
+# tests[id][:resource_cmd] - 'puppet resource' command to use with test_resource
+# tests[id][:show_pattern] - array of regexp patterns to use with test_show_cmd
+# tests[id][:ensure] - (Optional) set to :present or :absent before calling
+# tests[id][:code] - (Optional) override the default exit code in some tests.
+#
+# These keys are local use only and not used by test_harness_common:
+#
+# tests[id][:manifest_props] - This is essentially a master list of properties
+#   that permits re-use of the properties for both :present and :absent testing
+#   without destroying the list
+# tests[id][:resource_props] - This is essentially a master hash of properties
+#   that permits re-use of the properties for both :present and :absent testing
+#   without destroying the hash
+# tests[id][:title_pattern] - (Optional) defines the manifest title.
+#   Can be used with :af for mixed title/af testing. If mixing, :af values will
+#   be merged with title values and override any duplicates. If omitted,
+#   :title_pattern will be set to 'id'.
+#
+
+tests['preclean'] = {
+  manifest_props: '',
+  code:           [0, 2],
+  resource_props: {},
+}
+
+tests['default_properties'] = {
+  manifest_props: "
+    dup_host_ip_addr_detection_host_moves     => 'default',
+    dup_host_ip_addr_detection_timeout        => 'default',
+    anycast_gateway_mac                       => 'default',
+    dup_host_mac_detection_host_moves         => 'default',
+    dup_host_mac_detection_timeout            => 'default',
+  ",
+  resource_props: {
+    'dup_host_ip_addr_detection_host_moves' => '5',
+    'dup_host_ip_addr_detection_timeout'    => '180',
+    'dup_host_mac_detection_host_moves'     => '5',
+    'dup_host_mac_detection_timeout'        => '180',
+  },
+}
+
+tests['non_default_properties'] = {
+  manifest_props: "
+    anycast_gateway_mac                       => '1234.3456.5678',
+    dup_host_ip_addr_detection_host_moves     => '200',
+    dup_host_ip_addr_detection_timeout        => '20',
+    dup_host_mac_detection_host_moves         => '200',
+    dup_host_mac_detection_timeout            => '20',
+  ",
+  resource_props: {
+    'anycast_gateway_mac'                   => '1234.3456.5678',
+    'dup_host_ip_addr_detection_host_moves' => '200',
+    'dup_host_ip_addr_detection_timeout'    => '20',
+    'dup_host_mac_detection_host_moves'     => '200',
+    'dup_host_mac_detection_timeout'        => '20',
+  },
+}
+
+#################################################################
+# HELPER FUNCTIONS
+#################################################################
+
+# Full command string for puppet resource command
+def puppet_resource_cmd
+  cmd = UtilityLib::PUPPET_BINPATH + 'resource cisco_vxlan_global'
+  UtilityLib.get_namespace_cmd(agent, cmd, options)
+end
+
+def build_manifest_vxlan_global(tests, id)
+  if tests[id][:ensure] == :absent
+    state = 'ensure => absent,'
+    tests[id][:resource] = {}
```

Comment:
well I used the template file where it is { }.

Adding tests[id][:resource] = { 'ensure' => 'absent' } gives me a failure.
Failed Tests Cases:
      Test Case vxlan_global/test_vxlan_global.rb reported: #<Beaker::DSL::Outcomes::FailTest: 
    TestStep :: Match (?-mix:ensure +=> +'?absent'?) :: FAIL
    >

Do i need to make a change elsewhere ?